contracts/examples/digital-cash/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+#![no_std]
+#![allow(unused_attributes)]
+
+use elrond_wasm::{imports, require, sc_error};
+imports!();
+
+mod deposit_info;
+use deposit_info::DepositInfo;
+
+pub const SECONDS_PER_ROUND: u64 = 6; 
+
+#[elrond_wasm_derive::contract(DepositImpl)]
+pub trait Deposit {
+	
+	#[init]
+	fn init(&self) {
+		let my_address: Address = self.get_caller();
+		self.set_owner(&my_address);
+	}
+
+	fn get_expiration_round(&self,valability: u64) -> u64{
+		let valability_rounds = valability / SECONDS_PER_ROUND;
+		
+		return self.get_block_round() + valability_rounds;
+	}
+
+	#[endpoint]
+	#[payable("*")]
+	fn fund(
+		&self, #[payment] payment: BigUint,
+		#[payment_token] token: TokenIdentifier,
+		address: Address,
+		valability: u64
+		) -> SCResult<()> {
+
+		require!(
+			 payment > 0,
+			 "amount must be greater than 0"
+		);
+
+		let deposit  = &DepositInfo{
+			amount : payment,
+			depositor_address : self.get_caller(),
+			expiration : self.get_expiration_round(valability),
+			token_name : token
```

Comment:
No need for extra space after the field names

Reply:
Done!